docs/split-inputs.md

```diff
@@ -122,13 +122,18 @@ of separators.
 ## Fuzzed Data Provider
 
 [FuzzedDataProvider] is a single-header C++ library that is helpful for
-splitting a fuzz input into multiple parts of various types. Both advantage and
-disadvantage of using this library is that the input splitting happens
-dynamically, i.e. you don't need to define any structure of the input. This
-might be very helpful in certain cases, but would also make the corpus to be no
-longer in a particular format. For example, if you fuzz an image parser and
-split the fuzz input into several parts, the corpus elements will no longer be
-valid image files.
+splitting a fuzz input into multiple parts of various types. It is a part of
+LLVM and can be included via `#include <fuzzer/FuzzedDataProvider.h>` directive.
+If your compiler doesn't have this header (in case it's an older Clang version
+or some other compiler), you can copy the header from [here] and add it to your
+project manually. It should just work, as the header doesn't depend on LLVM.
+
+Both advantage and disadvantage of using this library is that the input
+splitting happens dynamically, i.e. you don't need to define any structure of
+the input. This might be very helpful in certain cases, but would also make the
+corpus to be no longer in a particular format. For example, if you fuzz an image
+parser and split the fuzz input into several parts, the corpus elements will no
+longer be valid image files.
```

Comment:
Add: "And you won't be able to simply add image files to your corpus".

Do you think it's worth mentioning how:
1. Changing your structure invalidates old corpus.
2. LPM solves some of these problems but forces you to define a structure up front..

Reply:
Added, thanks!

These are valid points, but I frankly don't want to overload this text too much. Plus, I hope that most people don't update the existing targets too often, and hopefully this point about corpus invalidation does not happen in most of the cases. But even if someone updates their fuzz target, it may happen once or twice, not every other week, so it's okay.